lib/omnibus/packagers/bff.rb

```diff
@@ -173,16 +173,16 @@ def create_bff_file
       # This implies that if we are in /tmp/staging/project/dir/things,
       # we will chown from 'project' on, rather than 'project/dir', which leaves
       # project owned by the build user (which is incorrect)
-      shellout!("sudo chown -R 0:0 #{staging_dir}/#{project.install_dir.match(/^\/?(\w+)/)}")
+      shellout!("sudo chown -R 0:0 #{File.join(staging_dir, project.install_dir)}")
       log.info(log_key) { "Creating .bff file" }
 
       # Since we want the owner to be root, we need to sudo the mkinstallp
       # command, otherwise it will not have access to the previously chowned
       # directory.
-      shellout!("sudo /usr/sbin/mkinstallp -d #{staging_dir} -T #{staging_dir}/gen.template")
+      shellout!("sudo /usr/sbin/mkinstallp -d #{staging_dir} -T #{File.join(staging_dir, 'gen.template')}")
```

Comment:
I don't really understand how only changing the staging_dir to a File.join fixes anything? You still have a forward slash in the mkinstallp path.